File: Unidad 4/p13_u4.cs
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            float[][] temp = new float[5][];
            for (int f = 0; f < 5; f++)
            {
                temp[f] = new float[7];
                for (int c = 0; c < 7; c++)
                {
                    Console.WriteLine("Escribe temperatura en (" + f + ", " + c + ")");
                    
                    temp[f][c] = float.Parse(Console.ReadLine());
                }
            }
            float[][] TempF = new float[5][];
            for (int f = 0; f < 5; f++)
            {
                TempF[f] = new float[7];
                for (int c = 0; c < 7; c++)
                {
                    TempF[f][c] = (temp[f][c] * 9 / 5) + 32;
                    Console.Write("Grados Farenheit en (" + f + ", " + c + ")" + TempF[f][c] + ",");
                }
                Console.WriteLine();
            }
        }
    }
}

File: Unidad 4/P13_u4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p13_u4;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class P13_u4 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        float[][] temp = new float[5][7];
        for (int f=0; f<5; f++){
           for (int c=0; c<7;c++){
               System.out.println("Escribe temperatura en ("+f+", "+c+")");
               temp[f][c]= sc.nextFloat();
        }
     }
     float [][] TempF=new float [5][7];
     for (int f=0; f<5;f++){
       for (int c=0; c<7;c++){
           TempF[f][c]=(temp[f][c]*9/5)+32;
           System.out.print("Grados Farenheit en ("+f+", "+c+")"+TempF[f][c]+",");
       }
       System.out.println();
     }
    }
}
    

